deepchem/feat/materials_featurizers.py

```diff
@@ -78,7 +78,7 @@ def _featurize(self, composition):
     except:
       feats = []
 
-    return np.array(feats)
+    return np.nan_to_num(np.array(feats))
```

Comment:
Could you add a note in the docs that NaNs will be converted to nums so users know about our defaults?

Reply:
Will do!